### PR TITLE
Show Config option

### PR DESCRIPTION
**Pull request by anaxceron**
Doc changes for [Issue 2569](https://github.com/zowe/docs-site/issues/2569):

- Added **Resolving Invalid Profiles** section to [Troubleshooting Zowe Explorer](https://docs.zowe.org/stable/troubleshoot/ze/troubleshoot-ze)

- Updated [Visual Studio Code (VS Code) Extension for Zowe](https://docs.zowe.org/stable/user-guide/ze-install/#configuring-zowe-explorer) article to reflect current doc standards/style guide.

Note: This is to be included in the v2.6 release documentation.

### CODE REVIEW

File: docs/user-guide/ze-install.md [hunk 1]
```diff
@@ -54,41 +55,69 @@ You can also watch the following videos to learn how to get started with Zowe Ex
...
-        "folderPath": "/path/to/directory"
-      }
-    ```
+3. Edit the settings in the file as needed.
+4. Save the file to keep changes.
 
-  where **/path/to/directory** is the folder location that you specify.
-  
-  3. Save the settings.
+    ![Configure Zowe settings](../images/ze/ZE-Configuration.gif)
```
Comment (traeok) [inline docs/user-guide/ze-install.md:83]: I noticed that this GIF shows an outdated version of ZE extension settings (before we used the "." separator for extension settings) - should the GIF stay as-is or should it be updated to reflect those changes?

File: docs/user-guide/ze-install.md [hunk 2]
```diff
@@ -54,41 +55,69 @@ You can also watch the following videos to learn how to get started with Zowe Ex
...
+### Modifying temporary file location settings
+
+Change the default folder location where temporary files are stored with the following steps:
+
+   1. Navigate to Zowe Explorer settings.
+   2. Under the data set, USS, or jobs settings that you want to edit, click the **Edit in settings.json** link.
+   3. Modify the following definition in the file:
+
+      ```json
+      "Zowe-Temp-Folder-Location": {
```
Comment (traeok) [inline docs/user-guide/ze-install.md:94]: I believe this is now "zowe.files.temporaryDownloadsFolder"

**Review by traeok — APPROVED** [review]

**Review by traeok — APPROVED** [review]
LGTM! Thanks for updating the GIF 😄

File: docs/user-guide/ze-install.md [hunk 3]
```diff
@@ -3,46 +3,47 @@
...
+- Get access to z/OSMF.
+- Install [Node.js](https://nodejs.org/en/download/) v8.0 or later.
+- Install [Visual Studio Code](https://code.visualstudio.com/).
+- Configure TSO/E address space services, z/OS data set, file REST interface, and z/OS jobs REST interface. For more information, see [z/OS Requirements](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf#z-os-requirements).
+- Create a Zowe CLI `zosmf` profile so that the extension can communicate with the mainframe.
 
 ### Profile notes:
 
-   - You can use your existing Zowe CLI `zosmf` profiles that are created with the Zowe CLI v.2.0.0 or later.
+- Use existing Zowe CLI `zosmf` profiles created with Zowe CLI v.2.0.0 or later.
```
Comment (zFernand0) [inline docs/user-guide/ze-install.md:29]: Personally, I will keep the suggestive wording, as we don't want to give people the idea that they have to use profiles created from the CLI

File: docs/user-guide/ze-install.md [hunk 4]
```diff
@@ -3,46 +3,47 @@
...
 Use the following steps to install Zowe Explorer:
 
 1. Address [the software requirements](#software-requirements).
-2. Open VSCode, and navigate to the **Extensions** tab on the left-hand side of the UI.
-3. Type **Zowe Explorer** in the search field.
+2. Open Visual Studio Code, and navigate to the **Extensions** tab on the **Activity Bar**.
+3. Type `Zowe Explorer` in the **Search** field.
   
-   Zowe Explorer appears in the list of extensions in the left-hand panel.
+   Zowe Explorer appears in the list of extensions in the **Side Bar**.
```
Comment (zFernand0) [inline docs/user-guide/ze-install.md:43]: Thanks for using proper VS Code terminology
🥳

File: docs/user-guide/ze-install.md [hunk 5]
```diff
@@ -54,41 +55,69 @@ You can also watch the following videos to learn how to get started with Zowe Ex
...
-  
-  3. Save the settings.
+    ![Configure Zowe settings](../images/ze/ZE-Configuration2.gif)
+
+### Modifying temporary file location settings
+
+Change the default folder location where temporary files are stored with the following steps:
+
+   1. Navigate to Zowe Explorer settings.
+   2. Under the data set, USS, or jobs settings that you want to edit, click the **Edit in settings.json** link.
```
Comment (zFernand0) [inline docs/user-guide/ze-install.md:90]: You can also click on the top-right corner icon in the VS Code settings.
image

However, navigating to the extension settings will try to open the `settings.json` file in context 😋

**Review by zFernand0 — COMMENTED** [review]
LGTM! 
I left one small comment about `use` vs `you can use` 😋

File: docs/user-guide/ze-install.md (hunk 3 above)
Comment (anaxceron) [reply docs/user-guide/ze-install.md:29]: Updated per your comment

**Pull request merged**